app/routes/docs.notices.producersS.mdx

```diff
@@ -0,0 +1,100 @@
+---
```

Comment:
Why is there this new file with the suffix S.mdx?

Reply:
I didn't find the way how to hide things new things in the file using feature flag, still working on it.